### ISSUE TITLE
OSCI-6737 - make spdx license checking configurable

### ISSUE THREAD

**Issue by jankaluza** [rpminspect/rpminspect#1378]
**Is your feature request related to a problem? Please describe.**
New rpminspect is now by default checks the spdx operators. This was introduced in https://github.com/rpminspect/rpminspect/issues/1335.

The issue is that checking some older releases is failing now because of that and we have no intention to use spdx on these old releases. See https://issues.redhat.com/browse/OSCI-6737 for more information.

**Describe the solution you'd like**

Make the spdx checks configurable, so we can disable them in rpminspect configuration file for these older releases.

**Comment by dcantrell** [comment]
Yeah, I was thinking about this recently.  I think adding a configuration option to tell the license inspection to use only the legacy names is necessary for a while.  At least until all of the older releases are retired.

**Comment by dcantrell** [comment]
The problem is in the libs package the License tag says "LGPLv2 and zlib".  "zlib" is the legacy license name and "Zlib" is the SPDX name.  But since SPDX abbreviations can be case-insensitive, rpminspect treats this as an SPDX abbreviation so now it thinks the license tag is an SPDX expression.

This should be easy to fix.
